test/unit/router-test.ts

```diff
@@ -63,3 +63,38 @@ test('Router > does not attempt to serialize when action.serializer = false', as
   req.write('{}');
   await router.handle(<any>req, <any>new MockResponse());
 });
+
+test('Router > #urlFor works with string argument', (t) => {
+  let container = new Container();
+  let logger = new Logger();
+
+  container.register('action:index', class TestAction extends Action {
+    serializer: false = false;
```

Comment:
This `: false` type isn't necessary - TS will infer the type since it's set explicitly.